### PR TITLE
fix instruction XML render bug

### PR DESCRIPTION
**Pull request by toddmedema**
Detect and render instruction nodes, including preceding if statements, ie `> {{ hi == true }} instruction`

### CODE REVIEW

**Comment by toddmedema** [comment]
Merging for progress; should review later

File: app/parsing/render/BlockRenderer.tsx [hunk 1]
```diff
@@ -320,6 +326,19 @@ export class BlockRenderer {
     };
   }
 
+  private extractInstruction(line: string): {text: string, visible: string} {
+    // Breakdown:
+    // \*\s*                    Match ">" and any number of spaces (greedy)
```
Comment (smartin015) [inline app/parsing/render/BlockRenderer.tsx]: Looks like the left half of this comment wasn't updated

Comment (smartin015) [inline app/parsing/render/BlockRenderer.tsx]: Also... I'm sensing a pattern here. Filed #69 to dedupe these methods.

**Comment by toddmedema** [comment]
@smartin015 filed an issue to discuss and implement the syntax for multi-line instructions: https://github.com/Fabricate-IO/expedition-quest-ide/issues/62 (out of scope for tonight)

**Pull request merged**